docs/source/production.rst

```diff
@@ -21,34 +80,34 @@ more.
 
 .. _diff-quality:
 
-Using SQLFluff on changes using `diff-quality`
-----------------------------------------------
+Using SQLFluff on changes using ``diff-quality``
+------------------------------------------------
```

Comment:
I think your IDE is thrashing with a change made recently by @tunetheweb . I think we want single backticks not double?

Reply:
for sphinx (rst) - I think we **do** want the double ones. Single ones seem to make the text _italic_ in the docs, double ticks make is `code`.